_layouts/post.html

```diff
@@ -38,6 +38,10 @@ <h1 class="post-big-title p-name" itemprop="name headline">{{ page.title | escap
                         </div>
                         <div class="post-content">
                             {{ content }}
+                            <div id="myModal" class="modal">
+                                <i id="modal-close" class="fa fa-times close" aria-hidden="true" onclick="closeModal()"></i>
+                                <img class="modal-content" id="img01">
```

Comment:
ten id na sztywno mi się tylko nie podoba

Reply:
Potrzebowałem do skryptu, żeby w niego wycelować przez getElementById

Reply:
a nie szło po tagu w obrębie modala? skoro modal i tak zawsze zawiera jeden img

Reply:
albo chociaż id zmienić np. na "modal-img", bo taki img01 dziwnie wygląda :)

Reply:
A można, jak najbardziej. Jeszcze jak :D Źle spojrzałem i myślałem, że komentarz jest do wcześniejszego ID

Reply:
to ode mnie plusik